packages/twenty-front/src/modules/settings/serverless-functions/components/SettingsServerlessFunctionsTable.tsx

```diff
@@ -20,6 +26,20 @@ const StyledTableBody = styled(TableBody)`
 
 export const SettingsServerlessFunctionsTable = () => {
   const { serverlessFunctions } = useGetManyServerlessFunctions();
+  const navigate = useNavigate();
+  const setHotkeyScope = useSetHotkeyScope();
+
+  useEffect(() => {
```

Comment:
I would use the same implementation as in the Modal component :  

```tsx
 const {
    goBackToPreviousHotkeyScope,
    setHotkeyScopeAndMemorizePreviousScope,
  } = usePreviousHotkeyScope();

  useEffect(() => {
    setHotkeyScopeAndMemorizePreviousScope(hotkeyScope);
    return () => {
      goBackToPreviousHotkeyScope();
    };
  }, [
    hotkeyScope,
    setHotkeyScopeAndMemorizePreviousScope,
    goBackToPreviousHotkeyScope,
  ]);

  useScopedHotkeys(
    [Key.Enter],
    () => {
      onEnter?.();
    },
    hotkeyScope,
  );
```

Reply:
You can try to use this implementation everywhere you meant to add a hotkey listener, it should work.

Reply:
I think it could even be abstracted in a hook useHotkeyScopeOnMount or something like that ?